core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphqlCallAction.kt

```diff
@@ -0,0 +1,36 @@
+package org.evomaster.core.problem.graphql
+
+import org.evomaster.core.problem.graphql.param.GraphqlParam
+import org.evomaster.core.problem.rest.GraphqlAction
+import org.evomaster.core.search.Action
+import org.evomaster.core.search.gene.Gene
+
+class GraphqlCallAction (
+        /**
+         * Identifier unique within the individual
+         * **/
+        val id: String,
+        val functionName: String,
+        val parameters:  MutableList<GraphqlParam>,
+        val operationType: String
```

Comment:
provide /** */ comment stating what operationType is.
furthermore, if the types are limited, could rather be an enum